jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -69,6 +69,14 @@ define([
     }
 
     PolymerSupport.init();
+    
+    // register delegate handlers that persist across dashboard mode changes  
+    $('#notebook_panel').on('change', '.show-code-label input', function() {
+        $('#notebook-container').toggleClass('hide-code', !this.checked);
+    });
+    $('#notebook_panel').on('click', '.more-info-link', function() {
+        $(this).parents('.help-area').toggleClass('show-more', 200);
+    });
```

Comment:
> 3. Before calling $helpArea.remove(), call .off('.DashboardHelpArea') to remove all event handlers

But you have to know exactly what node to call off on, which means you still need to know the structure of the template right?